app/src/main/res/layout/activity_quiz_question.xml

```diff
@@ -48,8 +47,8 @@
                 android:layout_height="wrap_content"
                 android:layout_weight="1"
                 android:indeterminate="false"
-                android:max="5"
-                android:minHeight="50dp"
+                android:max="10"
+                android:minHeight="30dp"
                 android:progress="0" />
 
             <TextView
```

Comment:
for `android:id` try to have consistency - e.g. your file is called `activity_quiz_question`, so it's a good practice to name all elements using that prefix, e.g. `progressBar` would be `android:id="@+id/activity_quiz_question_progress_bar"`,
`android:id="@+id/activity_quiz_question_progress_bar_textview"` or `android:id="@+id/activity_quiz_question_progress_bar_title"`